test-parameterization/src/Tailwind/Utilities.elm

```diff
@@ -377,17 +378,17 @@ globalStyles =
     ]
 
 
-bg : String -> Css.Style
+bg : Color -> Css.Style
 bg color =
     Css.batch
         [ Css.property "--tw-bg-opacity" "1"
-        , Css.property "background-color" ("rgb(" ++ color ++ " / var(--tw-bg-opacity))")
+        , Tailwind.Theme.toProperty "background-color" color
```

Comment:
I'm not sure what the best way to pass along this opacity is with this `Tailwind.Theme` module.

I was thinking something like this could be viable:

```elm
Tailwind.Theme.toProperty "background-color" "bg" color
```

```elm
module Tailwind.Theme exposing (..)

type Color = Color String (Maybe Opacity)

type Opacity = Opacity String

opacityOver25 : Opacity
opacityOver25 = Opacity "0.25"

withOpacity : Opacity -> Color -> Color
withOpacity (Opacity opacity) (Color color _) =
    Color color (Just opacity)

toProperty : String -> Color -> Css.Style
toProperty propertyName kind (Color color maybeOpacity) =
    let
        opacity =
            maybeOpacity
            |> Maybe.withDefault ("var(--tw-" ++ kind ++ "-opacity")
    in
    Css.property propertyName ("rgb(" ++ color ++ " / " ++ opacity + + ")")
```

Reply:
Also I'm not sure the best way to approach the opacity here with the way that the theme's colors come through from the Tailwind Config. They're String values which are mostly Hex. But that doesn't support adding these opacity values with the `/ ...` syntax.